### DISCUSSION TITLE
Unexpected MR title when using grouping

### DISCUSSION THREAD

**Discussion by itineric** [renovatebot/renovate#20433] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

The PR should take the defined groupName as title.

### What you tried so far.

Hi,

I am not getting the expected merge request title when using packageRule to have all Dockerfile modifications together.

My renovate.json
```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "docker":
  {
    "pinDigests": true
  },
  "regexManagers":
  [
    {
      "fileMatch": ["^Dockerfile$"],
      "matchStrings":
      [
        "datasource=(?<datasource>.*?) depName=(?<depName>.*?)( versioning=(?<versioning>.*?))?\\s(ARG|ENV) .*?_VERSION=(?<currentValue>.*?)\\s"
      ]
    }
  ],
  "packageRules":
  [
    {
      "matchUpdateTypes": ["major"],
      "enabled": false
    },
    {
      "matchPaths": ["**/Dockerfile"],
      "groupName": "docker dependencies",
      "groupSlug": "docker"
    }
  ]
}
```

The dumb Dockerfile I use for testing 

```
FROM node:14

# renovate: datasource=maven depName=org.apache.ant:ant
ARG ANT_VERSION=1.10.12
```

I only get one MR with the 2 modifications (add pin on FROM and upgrade ant version). The branch has the expected name renovate/docker but the title is "Pin dependencies", I was expecting something with "docker dependencies" as it is the defined "groupName".

Am I missing something here ?

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
Pin PRs are special and always use "Pin dependencies"
